<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#27292F</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>405</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Graphie</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F3EF52;</string>
     </property>
     <property name="text">
      <string>Options Rifle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F3EF52;</string>
     </property>
     <property name="text">
      <string>Login To Zerodha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_angel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F3EF52;</string>
     </property>
     <property name="text">
      <string>Login To Angel One</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_login_zerodha_api">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>405</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_process">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>179</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_zerodha_api">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>125</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F3EF52;</string>
     </property>
     <property name="text">
      <string>Login To Zerodha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_apikey">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>40</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>23</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:gray;</string>
     </property>
     <property name="text">
      <string>API key</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>64</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:gray;</string>
     </property>
     <property name="text">
      <string>API secret</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_api_secret">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>83</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
